docs-src/concepts/what-is-a-schedule.md

```diff
@@ -203,9 +203,9 @@ It is important to note that the [status](/concepts/what-is-a-workflow-execution
 
 :::caution
 
-A scheduled workflow may complete with a result up to the maximum blob size (2 MiB by default).
+A scheduled Workflow Execution may complete with a result up to the maximum blob size (2 MiB by default).
 However, due to internal limitations, results that are within 1 KiB of this limit cannot be passed to the next execution.
-So, for example, a workflow that returns a result of size above 2,096,127 (2MiB - 1KiB) bytes will be allowed to compete successfully, but that value will not be available as a last completion result.
+So, for example, a Workflow Execution that returns a result of size 2.1MiB (which is above 2MiB - 1KiB limit) will be allowed to compete successfully, but that value will not be available as a last completion result.
```

Comment:
No, a result of size 2.1MiB would not be allowed to complete since it's above the blob size limit. The whole point is we're warning about the tiny window in between the blob size limit and the "last completion" limit.